rumydata/_base.py

```diff
@@ -239,6 +239,7 @@ def _flatten_exceptions(cls, error):
         continuing to recurse until all errors have been yielded.
         """
         yield error
-        for el in error._errors:
-            for x in cls._flatten_exceptions(el):
-                yield x
+        if issubclass(type(error), UrNotMyDataError):
```

Comment:
I'm guessing this is to prevent passing of `None` from blowing up the iterator. If that's the case, then I would flip this to explicitly avoid none, as that makes the intent a little more clear.

```suggestion
        if error is not None:
```